src/cyhy_config/cyhy_config.py

```diff
@@ -0,0 +1,183 @@
+"""Finds a CyHy configuration.
+
+This module provides a function to find a CyHy configuration file.
+
+It will search in the following locations:
+ - A location passed in as a parameter
+ - Environment variables for an SSM parameter path
+ - Environment variables for a file path
+ - The current working directory
+ - The user's home directory
+ - The system's /etc directory
```

Comment:
Minor verbiage tweaks:
```suggestion
It will search in the following locations, in this order:
 - A location passed in as a parameter
 - Environment variable for an SSM parameter path
 - Environment variable for a file path
 - The current working directory
 - The user's home directory
 - The system's /etc directory
```

Reply:
I think that this should be an ordered list, e.g.:
```
1. A location passed in as a parameter
1. Environment variable for an SSM parameter path
```
and so on.

Reply:
See:
- [c88c22b](https://github.com/cisagov/cyhy-config/pull/5/commits/c88c22b6c5b4c8dc69b4cb8501bf0f2f0494b206)

Reply:
LGTM, thanks.